src/vscode-extras.ts

```diff
@@ -8,12 +8,12 @@ function getOpenUrl(textDocumentUri: URL): URL {
     const basePath: unknown = sourcegraph.configuration.get().value['vscode.open.basePath']
     if (typeof basePath !== 'string') {
         throw new Error(
-            'Setting `vscode.open.basePath` must be set in your [user settings](/user/settings) to open files in VS Code.'
+            `Setting \`vscode.open.basePath\` must be set in your [user settings](${sourcegraph.internal.sourcegraphURL.href}user/settings) to open files in VS Code.`
```

Comment:
**suggestion**: using the `URL` base resolve API to to guard against trailing slashes
```suggestion
            `Setting \`vscode.open.basePath\` must be set in your [user settings](${new URL('/user/settings', sourcegraph.internal.sourcegraphURL.href).href}) to open files in VS Code.`
```